pipeline/transformer/transformer_mets/src/main/scala/weco/pipeline/transformer/mets/transformer/ImageUtils.scala

```diff
@@ -27,19 +27,24 @@ object ImageUtils {
     )
 
   private final val thumbnailDim = "200"
+  private final val thumbnailPathSuffix =
+    s"full/!$thumbnailDim,$thumbnailDim/0/default.jpg"
 
-  def buildThumbnailUrl(bnumber: String,
-                        validThumbnailFile: FileReference): Option[String] =
+  // The /thumbs URL is routed to DLCS which handles only images
+  // other asset types are routed to the iiif-builder service at /thumb
```

Comment:
Worth linking to the discussions/documentation on this topic? So people can dig into it if they're modifying it, and don't think it's overly simple.